### ISSUE TITLE
Hosted version is down, https://www.chatwithgpt.ai

### ISSUE THREAD

**Issue by kushie0** [cogentapps/chat-with-gpt#192]
Any chance of returning it?

**Comment by devyujie** [comment]
![image](https://github.com/cogentapps/chat-with-gpt/assets/48620527/c913cf60-6b30-4c12-8c96-3b18f6bcf6d4)

**Comment by lucasfariaslf** [comment]
Looking at the code frequency and last commit, I have a feeling this project is either dead or on halt. I also don't wanna host my own server, in the meantime I'm using this: https://github.com/mckaywrigley/chatbot-ui

**Comment by josephfinlayson** [comment]
This is very sad. I used it every day!

**Comment by kushie0** [comment]
i was also a frequent user. i also started using chatbot-ui but i want to at least recover my data from this app, haven't looked into it yet.

**Comment by smadep** [comment]
that's bad :-( I prefered it to the official one, because I could tweak the UI to fit my needs.

**Comment by leoffx** [comment]
I'm now hosting it here https://chatwithgpt-app.web.app/

I'm not hosting any server part, also because of privacy concerns, I don't want to hold any of your data. Because of this some parts might be broken (e.g login?)

Feel free to contribute to the fork https://github.com/leoffx/chat-with-gpt , I already added the new models to it

**Comment by lucasfariaslf** [comment]
@leoffx You're a champ! Was about to fork this to add the new models. I can't let go of this UI, it's perfectly minimalistic.

**Comment by msikora78** [comment]
@leoffx Are you planning on implementing image support for the Vision model?

**Comment by leoffx** [comment]
@msikora78 not really, but if anyone contributes with a PR, I can review and deploy it

**Comment by kushie0** [comment]
> I'm now hosting it here https://chatwithgpt-app.web.app/
> 
> I'm not hosting any server part, also because of privacy concerns, I don't want to hold any of your data. Because of this some parts might be broken (e.g login?)
> 
> Feel free to contribute to the fork https://github.com/leoffx/chat-with-gpt , I already added the new models to it

This isn't working for me. Are you self-hosting?

**Comment by leoffx** [comment]
@kushie0 what is not working? As I said, I'm not hosting any server component, only the frontend

**Comment by kushie0** [comment]
@leoffx nothing loads after i insert my api key and send a message

**Comment by leoffx** [comment]
@kushie0 are there errors in the console? create an issue in my repo with most info you can add (browser, error message, how to reproduce) so we don't flood it here

**Comment by kushie0** [comment]
@leoffx I don't think you have issues enabled in your repo :) but yes I would love to do all that instead of flooding this
